### PR TITLE
Deploy AggregationISM to mainnet2

### LINKED ISSUE #2479: Default to 1/2 aggregation of MessageIdISM and MerkleProofISM in mainnet2
Assess quorum of validators moving to DTM first

### PR DESCRIPTION
**Pull request by aroralanuk**
### Description

- Deployed aggregationISM to mainnet2 (context - hyperlane).
- Set as defaultISM for mainnet2

### Drive-by changes

None

### Related issues

Fixes #2479 for context hyperlane

### Backward compatibility

Backward compatible? Yes

### Testing

Manual

### CODE REVIEW

File: typescript/sdk/src/ism/HyperlaneIsmFactory.ts [hunk 1]
```diff
@@ -151,19 +151,10 @@ export class HyperlaneIsmFactory extends HyperlaneApp<IsmFactoryFactories> {
...
-        const ism = await this.deploy(chain, config.domains[origin], origin);
-        isms[origin] = ism.address;
-      }),
-    ).then((results) => {
-      results.forEach((result) => {
-        if (result.status === 'rejected') {
-          this.logger(`Failed to deploy routing ISM: ${result.reason}`);
-        }
-      });
-    });
```
Comment (jmrossy) [inline typescript/sdk/src/ism/HyperlaneIsmFactory.ts:166]: Can you confirm we don't need to check for these failure statuses anymore? Will the deploy method throw if there's an issue?

File: typescript/sdk/src/consts/environments/test.json
Comment (yorhodes) [inline typescript/sdk/src/consts/environments/test.json:1]: I don't think checking this in is useful

File: typescript/sdk/src/ism/HyperlaneIsmFactory.ts [hunk 2]
```diff
@@ -371,6 +362,7 @@ export async function moduleMatchesConfig(
   origin?: ChainName,
 ): Promise<boolean> {
   const provider = multiProvider.getProvider(chain);
+  if (moduleAddress === ethers.constants.AddressZero) return false;
```
Comment (yorhodes) [inline typescript/sdk/src/ism/HyperlaneIsmFactory.ts:365]: I don't think this is correct

File: typescript/sdk/src/ism/HyperlaneIsmFactory.ts (hunk 1 above)
Comment (aroralanuk) [reply typescript/sdk/src/ism/HyperlaneIsmFactory.ts:166]: the deploy method for each type of ISM does the error handling inside the `this.deploy` method.

File: typescript/sdk/src/ism/HyperlaneIsmFactory.ts (hunk 2 above)
Comment (aroralanuk) [reply typescript/sdk/src/ism/HyperlaneIsmFactory.ts]: Fixed!

**Review by yorhodes — APPROVED** [review]

**Pull request merged**